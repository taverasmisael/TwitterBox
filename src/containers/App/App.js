// @flow
import React from 'react'
import TwitterBox from '../../components/TwitterBox'

const App = () => <TwitterBox />

export default App
