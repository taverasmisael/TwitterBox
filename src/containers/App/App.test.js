import React from 'react'
import { render } from 'react-testing-library'
import App from './App'

describe('containers/App', () => {
  it('renders without crashing', () => {
    const { getByText } = render(<App />)
    expect(getByText('Twitter Box Example')).toBeInTheDocument()
  })

  it('matches snapshot', () => {
    const { container } = render(<App />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
