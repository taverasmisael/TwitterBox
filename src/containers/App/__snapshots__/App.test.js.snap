// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/App matches snapshot 1`] = `
<h1>
  Twitter Box Example
</h1>
`;
